src/react/OverflowItem.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { useMergedRefs } from "@fluentui/react-northstar";
+import { useOverflowItem } from './useOverflowItem';
+
+export type OverflowItemChildProps = React.HTMLAttributes<HTMLElement> & {
+  ref?: React.Ref<never>;
+};
+
+export type OverflowItemProps = {
+  id: string | number;
+  groupId?: string | number;
+  priority?: number;
+  children: React.ReactElement;
+};
+
+export const OverflowItem = React.forwardRef((props: OverflowItemProps, ref) => {
+  const { id, groupId, priority, children } = props;
+
+  const containerRef = useOverflowItem<HTMLDivElement>(id, priority, groupId);
+
+  const mergedRef = useMergedRefs(containerRef, ref);
+
+  return React.cloneElement(React.Children.only(children), { ref: mergedRef, id, groupId, priority });
```

Comment:
I didn't recursively clone like MenuTrigger/Tooltip.  It seems that those are deeper scenarios with checks if its a FluentTrigger.

Reply:
it does bring up an interesting question though, how does third party components that clone children work with Fluent ?